docs/reference/commandline/run.md

```diff
@@ -244,6 +244,23 @@ $ docker run -it --pid=container:my-redis my_strace_docker_image bash
 $ strace -p 1
 ```
 
+### <a name="uts"></a> UTS settings (--uts)
+
+```text
+--uts=""  : Set the UTS namespace mode for the container
+            'host': use the host's UTS namespace inside the container
+```
+
+The UTS namespace is for setting the hostname and the domain that's visible to
+running processes in that namespace. By default, all containers, including
+those with `--network=host`, have their own UTS namespace. The `host` setting
+will result in the container using the same UTS namespace as the host. Note
+that `--hostname` and `--domainname` are invalid in `host` UTS mode.
```

Comment:
Perhaps also consider rephrasing "Note that" (which is a "Note"). Perhaps it needs context (why!? because `--network=host` runs in the host's network namespace, and the container would otherwise attempt to change the hosts' configuration. I guess that's what the paragraph below was intended for, but they're now two parts of the same thing 🤔 

Maybe we need a separate heading/example example combining both (with some warnings added, because it's changing the host)

Reply:
carried in https://github.com/docker/cli/pull/4712